contracts/LifCrowdsale.sol

```diff
@@ -191,18 +210,41 @@ contract LifCrowdsale is Ownable, Pausable {
 
     uint256 tokens = weiSent.mul(privatePresaleRate);
 
-    require(totalPresaleWei.add(weiSent) <= maxPresaleWei);
```

Comment:
Why is this validation gone? I understand that we need to modify it to take the wei/USD rate into account, but not remove it. Or am I missing something?

Reply:
Because we dont know the `weiPerUSDinPresale` yet, so we cannot check that the amount of tokens added on private presale are less that certain amount, but since the transactions with this functions would be recorded and the investors can see the totalSupply before the public presale we dont need to check the amount, I mean, if we issue a wrong number of tokens it would be there on the totalSupply.

Reply:
Ok, I agree this validation is not needed in the private presale. But I thought this was for the public presale, the diff confused me, github's fault ;) and now I see that this validation is in the `validPresalePurchase` so all good :+1: